app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -180,20 +180,20 @@ def parse_openshift_image(openshift_image)
         new_result[:tag] ||= json["tag"] if json.keys.include?("tag")
 
         docker_metadata = openshift_image[:dockerImageMetadata]
+        docker_labels = parse_identifying_attributes(docker_metadata[:Config][:Labels], 'docker_labels', "openshift")
+        labels = parse_labels(openshift_image)
         new_result.merge!(
           :architecture          => docker_metadata[:Architecture],
           :author                => docker_metadata[:Author],
           :command               => docker_metadata[:Config][:Cmd],
           :entrypoint            => docker_metadata[:Config][:Entrypoint],
           :docker_version        => docker_metadata[:DockerVersion],
-          :exposed_ports         => parse_exposed_ports(
-            docker_metadata[:Config][:ExposedPorts]),
-          :environment_variables => parse_env_variables(
-            docker_metadata[:Config][:Env]),
+          :exposed_ports         => parse_exposed_ports(docker_metadata[:Config][:ExposedPorts]),
+          :environment_variables => parse_env_variables(docker_metadata[:Config][:Env]),
           :size                  => docker_metadata[:Size],
           :labels                => parse_labels(openshift_image),
-          :docker_labels         => parse_identifying_attributes(docker_metadata[:Config][:Labels],
-                                                                 'docker_labels', "openshift"),
+          :docker_labels         => docker_labels,
+          :tags                  => map_labels('ContainerImage', labels + docker_labels)
```

Comment:
@simon3z Im only merging them when sending them to be mapped into tags. This would allow users to turn docker labels into tags for `ContainerImages`. I think its unlikely that someone would set a rule on something that would be common for both docker-labels and kubernetes labels. 
@cben What would happen in the above case which they have the same key but different values?